src/components/Person/Person.jsx

```diff
@@ -2,0 +1,19 @@
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">
+      {person.age ? <p>I am ${person.age}</p> : null}
```

Comment:
The template literal syntax is incorrect here. It should be `{`I am ${person.age}`}` instead of `<p>I am ${person.age}</p>`.